service/service.go

```diff
@@ -20,6 +20,7 @@ import (
 	"github.com/giantswarm/azure-operator/client"
 	"github.com/giantswarm/azure-operator/flag"
 	"github.com/giantswarm/azure-operator/service/azureconfig"
+	azureconfigconfig "github.com/giantswarm/azure-operator/service/azureconfig/config"
```

Comment:
Now this is kind of annoying that it has to be aliased because `config` is a `New` parameter.

Any ideas for a more unique name? `configuration`, `settings`?

Reply:
What about `flags`? It actually maps information from the command line flags to the service.

Reply:
I thought about it as well but there is also `"github.com/giantswarm/azure-operator/flag"` import which would collide... Unless we make one plural and other singular, but I think we both agree that's awful.

I think `settings` makes sense as this is previous `// Settings.` part of the `Config`s.

Reply:
I do not mind. Don't have better ideas either.

Reply:
Also `settings` or `setting`?